<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwb_v1</class>
 <widget class="QWidget" name="pwb_v1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;高压模块调试&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>30</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Channel0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Channel1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Channel2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Channel3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameter State</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Channel3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_17"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_18"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_16"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Channel0                   </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Channel1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Channel2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>781</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Sensor</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Template                                                  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Humidity                                               </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>781</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Current  &amp;&amp; voltage</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Channel0</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>69</y>
          <width>355</width>
          <height>131</height>
         </rect>
        </property>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>0</y>
           <width>355</width>
           <height>131</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Channel1</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>69</y>
          <width>355</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>Channel3</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_11" native="true">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>69</y>
          <width>355</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>Channel2</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>69</y>
          <width>355</width>
          <height>131</height>
         </rect>
        </property>
        <widget class="QWidget" name="widget_8" native="true">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>0</y>
           <width>355</width>
           <height>131</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>DAQ</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>&amp;Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>&amp;Save...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>11</y>
     <width>41</width>
     <height>50</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Id&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>771</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Connect</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RTU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TCP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASCII</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Address</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>483</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
